tests/cps-files/lib/cps/full.cps

```diff
@@ -0,0 +1,53 @@
+{
+    "name": "full",
+    "cps_version": "0.10.0",
+    "version": "1.0.1",
+    "components": {
+        "sample0": {
+            "type": "archive",
+            "compile_flags": [
+                "-fsentinal"
+            ],
+            "includes": {
+                "c": [
+                    "/err"
+                ]
+            },
+            "defines": {
+                "c": [
+                    "-DFAIL"
+                ]
+            },
+            "location": "fake"
+        },
+        "fullSample": {
+            "type": "archive",
+            "compile_flags": [
+                "-fopenmp"
```

Comment:
Not directly relevant, but I suspect OpenMP should be a [feature](https://cps-org.github.io/cps/features.html)...

Reply:
Makes sense to me, but I think `cps-config` does not yet support those features. What would be a more appropriate `compile_flag`?

Reply:
I replaced with `-fvectorize`. Let me know if this is a better choice!

Reply:
The goal of `compile_features` is to enumerate things like this for the sake of portability, so that you can still compile if using a "non-standard" compiler that spells the flags differently. Broadly speaking, if you are using `compile_flags`, someone is doing something wrong.

However, since here we're specifically testing `compile_flags`, it doesn't matter that much what flag we use. The original comment was more that, unrelated to this PR, the spec should probably learn to spell a bunch of these other features. (Eventually, at least.)

...and on that note, I wonder if Features should change somehow to include "suggested" spellings, so that a consumer that doesn't recognize a feature can use the suggested flag as a fallback with some hope that it will Just Work. (That, or tools should have a way for users to alter the feature-to-flag mapping so that an unknown but required feature doesn't force a tool upgrade.)

Reply:
Makes sense. Some official guidance on how people should spell certain things, before build tools are forced to understand that "openmp" "OpenMP" and "omp" are actually the same thing